UITests/UITests.Tests.Shared/Tests.cs

```diff
@@ -0,0 +1,117 @@
+    {
+        public static TestApplicationInfo WinUICsUWPSampleApp
+        {
+            get
+            {
+                string assemblyDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+                string baseDirectory = Path.Combine(Directory.GetParent(assemblyDir).Parent.Parent.Parent.Parent.FullName, "UITests.App");
+
+                Log.Comment($"Base Package Search Directory = \"{baseDirectory}\"");
+
+                var exclude = new[] { "Microsoft.NET.CoreRuntime", "Microsoft.VCLibs", "Microsoft.UI.Xaml", "Microsoft.NET.CoreFramework.Debug" };
+                var files = Directory.GetFiles(baseDirectory, "*.msix", SearchOption.AllDirectories).Where(f => !exclude.Any(Path.GetFileNameWithoutExtension(f).Contains));
+
+                if (files.Count() == 0)
+                {
+                    throw new Exception(string.Format("Failed to find '*.msix' in {0}'!", baseDirectory));
+                }
+
+                string mostRecentlyBuiltPackage = string.Empty;
+                DateTime timeMostRecentlyBuilt = DateTime.MinValue;
+
+                foreach (string file in files)
+                {
+                    DateTime fileWriteTime = File.GetLastWriteTime(file);
+
+                    if (fileWriteTime > timeMostRecentlyBuilt)
+                    {
+                        timeMostRecentlyBuilt = fileWriteTime;
+                        mostRecentlyBuiltPackage = file;
+                    }
+                }
+
+                return new TestApplicationInfo(
+                    "UITests.App",
+                    "3568ebdf-5b6b-4ddd-bb17-462d614ba50f_gspb8g6x97k2t!App",
+                    "3568ebdf-5b6b-4ddd-bb17-462d614ba50f_gspb8g6x97k2t",
+                    "UITests.App",
+                    "UITests.App.exe",
+                    mostRecentlyBuiltPackage.Replace(".msix", string.Empty),
+                    "24d62f3b13b8b9514ead9c4de48cc30f7cc6151d",
+                    baseDirectory);
+            }
+        }
+
+        public static TestSetupHelper.TestSetupHelperOptions TestSetupHelperOptions
+        {
+            get
+            {
+                return new TestSetupHelper.TestSetupHelperOptions
+                {
+                    AutomationIdOfSafeItemToClick = string.Empty
+                };
+            }
+        }
+
+        [ClassInitialize]
+        [TestProperty("RunAs", "User")]
+        [TestProperty("Classification", "ScenarioTestSuite")]
+        [TestProperty("Platform", "Any")]
+        public static void ClassInitialize(TestContext testContext)
+        {
+            TestEnvironment.Initialize(testContext, WinUICsUWPSampleApp);
+        }
+
+        [ClassCleanup]
+        public static void ClassCleanup()
+        {
+            TestEnvironment.AssemblyCleanupWorker(WinUICsUWPSampleApp);
+        }
+
+        [TestCleanup]
+        public void TestCleanup()
+        {
+        }
+
+        [TestMethod]
+        public void SimpleLaunchTest()
```

Comment:
Should we split this test out into it's own file so that it's easier for folks to just copy the file and start organizing tests?

Do we have a link to some docs on what actions/verifications are possible within these tests from the framework?

Reply:
I just realized too that the XAML for this test is being grabbed from the `MainPage.xaml`. It'd be cool if we could support multiple pages and then use an Attribute here or something to associate this test to a particular page? And then have the test app load the appropriate page first into a Frame and then execute the test?

Reply:
Ah cool, I saw @RosarioPulella had started looking at this as part of his PR #3512, see comparison here: https://github.com/windows-toolkit/WindowsCommunityToolkit/compare/muxtestinfra...RosarioPulella:test-TextBoxMask

Let's improve upon that in a separate PR then, and just get this one merged first then.

Reply:
Think this discussion is going to be followed-up in #3512.